README.md

```diff
@@ -3,32 +3,42 @@ Operator to automate workload identity setup on GCP clusters
 
 ## Introduction
 
-[Workload identity](https://cloud.google.com/iam/docs/workload-identity-federation) gives a workload app the ability to exchange an oauth token, acquired from some external identity provider, for a GCP access token.
+[Workload identity](https://cloud.google.com/iam/docs/workload-identity-federation) gives a workload app the ability to exchange an OAuth token, acquired from some external identity provider, for a GCP access token.
 The application can use this access token to use whatever resources it needs on GCP.
 
 
 > Workload Identity allows a Kubernetes service account in your GKE cluster to act as an IAM service account.
 > Pods that use the configured Kubernetes service account automatically authenticate as the IAM service account when accessing Google Cloud APIs.
 > Using Workload Identity allows you to assign distinct, fine-grained identities and authorization for each application in your cluster.
 
+## Concepts
+
+This operator is set up to work on a multi-cluster setup. What is a multi-cluster setup?
+
+> Multi-cluster Kubernetes is a Kubernetes deployment method that consists of two or more clusters.
+
+It operates on the notion of management and workload clusters. Where a management cluster manages the lifecycle of a workload cluster and vice versa.
```

Comment:
nitpick :grimacing: 

> Where a management cluster manages the lifecycle of a workload cluster and vice versa.

"and vice versa"? Would that mean that a workload cluster manages the lifecycle of a management cluster?

Reply:
😄 No, I mean't that a workload cluster has their lifecycle managed by a management cluster. 
I've updated it to be more akin to that found here - https://cluster-api.sigs.k8s.io/user/concepts.html